linode/resource_linode_user.go

```diff
@@ -59,6 +168,11 @@ func resourceLinodeUserCreate(ctx context.Context, d *schema.ResourceData, meta
 	if err != nil {
 		return diag.Errorf("failed to create user: %s", err)
 	}
+
+	if err := updateUserGrants(ctx, d, meta); err != nil {
```

Comment:
Could we make a function `hasGrantsConfigured` which takes in the `schema.ResourceData` and returns whether or not any of these told level attributes have been specificed? We don't want to make a call to update grants (here or in update) unless it's explicitly configured as users might want to configure Grants using cloud manager.